examples/quantization/README.md

```diff
@@ -60,6 +60,7 @@ Checkpoint saved in `output_dir` can be directly passed to `trtllm-build`.
 - qformat: Specify the quantization algorithm applied to the checkpoint.
     - nvfp4: Weights are quantized to NVFP4 block-wise with size 16. Activation global scale are calibrated.
     - fp8: Weights are quantized to FP8 tensor wise. Activation ranges are calibrated tensor wise.
+    - fp8_pc_pt: Weights are quantized to FP8 vector-wise. Activation ranges are calibrated tensor wise.
```

Comment:
is this activation per-tensor? the name `fp8_pc_pt` will conflict with modelopt's definition because the `pt` there means per-token https://github.com/NVIDIA/TensorRT-Model-Optimizer/blob/5a2bf34951d64375da1e76112af5a826e9f7c81c/examples/llm_ptq/hf_ptq.py#L66. 
but in the file change tensorrt_llm/quantization/quantize_by_modelopt.py below, I saw you're also referring the per-channel, per-token mode

Reply:
My understanding is that 'calibrated tensor wise' for activation means this is static quantization. The per-channel per-token quantization only applies to weights.

Reply:
Fixed with 520c27b